pkg/infra/metrics/metrics.go

```diff
@@ -116,6 +138,15 @@ var (
 
 	// MRenderingSummary is a metric summary for image rendering request duration
 	MRenderingSummary *prometheus.SummaryVec
+
+	// MAccessSummary is a metric summary for access request duration
+	MAccessSummary prometheus.Histogram
+
+	// MAccessSummary is a metric summary for access request duration
```

Comment:
```suggestion
	// MAccessPermissionsSummary is a metric summary for loading permissions request duration when evaluating access
```
Something like this?